bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java

```diff
@@ -503,17 +513,26 @@ public void run() {
                     // responses
                     for (int i = 0; i < requestsCount; i++) {
                         ForceWriteRequest req = localRequests.get(i);
+                        req.getForceWriteWaiters().forEach(ele -> {
```

Comment:
`req.process()` is already iterating over all the entries for each `ForceWriteRequest`. It would be better to reuse the same loop.

Additionally using `forEach` has 2 problems: 
 1. allocates an iterator, unlike doing `for (int i =0; i... .)` 
 2. the lambda is also capturing variables and thus it will allocate a temporary object